CMakeLists.txt

```diff
@@ -92,7 +92,6 @@ find_program(CCACHE_FOUND ccache)
 if(CCACHE_FOUND)
   SET_PROPERTY(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
   SET_PROPERTY(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
-  message(STATUS "Found ccache: will speed builds up with it")
```

Comment:
Not sure about not including it - it will be annoying when it breaks things and we forget it is/could be involved... 😦

Reply:
We'll learn fast and we won't get spammed about it all the other times it works fine!